### ISSUE TITLE
Graph Issue

### ISSUE THREAD

**Issue by shivam-MBZUAI** [spookyQubit/refreshExtractiveNeuralSummarizer#2]
How did you plot training curves in images folder? 

What does x-axis define? Epoch, batches, (what)

**Comment by spookyQubit** [comment]
Hi @confusedAI , its been long since I did this project but I will try my best to jog my memory to answer this. 

In my [training loop](https://github.com/spookyQubit/refreshExtractiveNeuralSummarizer/blob/master/main.py#L94) I am dumping the loss and other metrics which is what I am plotting. So the x axis should be number of training batches the model has seen during training.

**Comment by spookyQubit** [comment]
Closing it for now. -- Thanks.
